frontend/src/components/ProductListCard/ProductListCard.js

```diff
@@ -0,0 +1,183 @@
+            // }
+            return (
+                <Title level={4} className="price">
+                    Ab {product.smallestPrice}
+                    {/* Ab {lowestPrice} € */}
+                </Title>
+            );
+        }
+    }
+
+    getCardItem(product) {
+        const price = this.getLowestPrice(product);
+        return (
+            <Col span={8}>
+                <Card>
+                    {/* title={product.name} key={product._id} bordered={true} */}
+                    <Row gutter={8}>
+                        <Col span={10}>
+                            <Row id="product_image" align="middle">
+                                <img
+                                    src={product.images?.length > 0 ? product.images[0] : placeholder}
+                                    alt="bild"
+                                    width="100%"
+                                />
+                            </Row>
+                        </Col>
+                        <Col id="content_col" span={14}>
+                            <Row id="fairbundle_tag" justify="start">
+                                <Paragraph>
+                                    {product.hasFairbundle ? (
+                                        <Tag color="#78A262">Fairbundle</Tag>
+                                    ) : (
+                                        ""
+                                    )}
+                                </Paragraph>
+                            </Row>
+                            <Row id="product_title" justify="start">
+                                <Title level={4}>{product.name}</Title>
+                            </Row>
+                            <Row id="product_description" justify="start">
+                                <Paragraph>
+                                    <Text>{product.description}</Text>
+                                </Paragraph>
+                            </Row>
+                            <Row gutter={8} align="middle">
+                                <Col span={12}>{price}</Col>
+                                <Col span={12}>
+                                    <Link to={`/product/${product._id}`}>
+                                        <Button
+                                            shape="round"
+                                            size="middle"
+                                            type="primary"
+                                        >
+                                            Details
+                                        </Button>
+                                    </Link>
+                                </Col>
+                            </Row>
+                        </Col>
+                    </Row>
+                </Card>
+            </Col>
+        );
+    }
+
+    handleChange(checked) {
+        this.setState({checked});
+    }
+
+    handleOrdering(ordering) {
+        this.setState({ordering});
+    }
+
+    hasFairbundle(p) {
```

Comment:
Ist etwas missverständlich, da man bei `hasFairbundle` erwarten würde, dass ein `boolean` returned wird. Es wird aber die gefilterte Liste returned. Vermutlich muss die auch in den `state` gesetzt werden, damit man unten die Anzahl der Ergebnisse richtig anzeigen kann.